java-devcourse-project/src/classHomework7/sameNameClassContainer/SameName.java

```diff
@@ -0,0 +1,5 @@
+package classHomework7.sameNameClassContainer;
```

Comment:
Как обсуждали на уроке, сложные имена пакетов пишутся слитно маленькими буквами или же разделяются точками.
```suggestion
package classHomework7.same.name.class.container;
```